daemon/algod/api/algod.oas2.json

```diff
@@ -4141,20 +4141,35 @@
             "$ref": "#/definitions/SimulationOpcodeTraceUnit"
           }
         },
+        "approval-program-hash": {
+          "description": "Hash digest of the approval program executed in transaction.",
+          "type": "string",
+          "format": "bytes"
```

Comment:
I think the problem is this should be `byte`, _not_ `bytes`

Reply:
right, good catch, see [format byte](https://github.com/algorand/go-algorand/pull/5658/commits/28d226eddcc9ea2a4f6f828af5345258852c450d)